src/mixins/input-management/input-management.ts

```diff
@@ -34,6 +34,8 @@ export class InputManagement extends ModulVue
     public autocomplete: string;
     @Prop()
     public focus: boolean;
+    @Prop({ default: '' })
```

Comment:
Je n'ajouterais pas cette props dans la mixins, je déplacerais plutot vers m-textfield car dans certain composante qui utilise inputmanagement la selection de fait pas de sens et ne peut pas fonctionner (...vite vite comme ca je vois le ca dans le cas d'un m-timepicker ou d'un m-select)

Reply:
j'ai mis dans textfield: 
@Watch('selection')
    updateSelection(): void {
        changeSelection(this.as<InputState>().getInput() as HTMLInputElement, this.selection);
    }

J'ai extract ça dans une fonction si jamais on veut le réutiliser ailleurs.

J'ai aussi créé un interface InputSelectable si jamais les gens veulent l'utiliser